### DISCUSSION TITLE
Feature request - more detailed FP version information

### DISCUSSION THREAD

**Discussion by Lotus49b** [freeplane/freeplane#975] [Ideas]
I've been installing a number of new preview versions of FP lately while trying out the new layout functionality.

Sometimes I need to revert to previous versions, and I lose track of what version is currently installed.

The "About" information displays a version number but it's quite high level:
![image](https://user-images.githubusercontent.com/11365744/215677971-603bdc26-33d2-483c-96f4-3797df3c8af5.png)

It would be helpful to see the preview version information too, or even the filename that it was installed from:
![image](https://user-images.githubusercontent.com/11365744/215678231-be20a02f-cecc-4a61-95ec-805c3abbeba4.png)

Thanks

**Comment by euu2021** [comment]
Issue #837
